pkg/status/templates/remoteconfig.tmpl

```diff
@@ -6,9 +6,13 @@ NOTE: Changes made to this template should be reflected on the following templat
 Remote Configuration
 ====================
     {{ with .remoteConfiguration }}
-    API Key: {{ if .apiKeyScoped }}Authorized{{ else }}Not authorized{{ end }}
-    Feature: {{ if .orgEnabled }}Enabled{{ else }}Disabled{{ end }}
+    {{ if not .disabledReason }}
+    Organization enabled: {{ if and .orgEnabled (eq .orgEnabled "true") }}True{{ else }}False{{ end }}
```

Comment:
`.orgEnabled` is a string that can be `"true"` or `"false"`. It's not a boolean since it's a direct copy of an `expvar` (there is no boolean expvar)
Problem is that `text/template` consider `{{ if .orgEnabled }}` to be true if `.orgEnabled = "false"`

Reply:
Makes sense, thanks 👍